cloudify_terminal/examples/cisco_flash_list.yaml

```diff
@@ -2,7 +2,7 @@ tosca_definitions_version: cloudify_dsl_1_3
 
 imports:
   - http://www.getcloudify.org/spec/cloudify/4.3.1/types.yaml
-  - https://raw.githubusercontent.com/cloudify-incubator/cloudify-utilities-plugin/1.5.3/plugin.yaml
+  - https://raw.githubusercontent.com/cloudify-incubator/cloudify-utilities-plugin/1.5.4/plugin.yaml
```

Comment:
`1.6.0`

Reply:
Updated code with correct fixes in 1.5.4